[openbox] Failed to install

```
checking for PANGO... no
configure: error: Package requirements (pango >= 1.8.0 pangoxft >= 1.8.0) were not met:

Package 'graphite2', required by 'harfbuzz', not found

Consider adjusting the PKG_CONFIG_PATH environment variable if you
installed software in a non-standard prefix.

Alternatively, you may set the environment variables PANGO_CFLAGS
and PANGO_LIBS to avoid the need to call pkg-config.
See the pkg-config man page for more details.
openbox failed to install: `./configure --prefix=/usr/local --libdir=/usr/local/lib64 --mandir=/usr/local/share/man --build=x86_64-cros-linux-gnu --host=x86_64-cros-linux-gnu --target=x86_64-cros-linux-gnu` exited with 1
```

--- a/packages/openbox.rb
+++ b/packages/openbox.rb
@@ -74,7 +74,7 @@ EOF"
   def self.postinstall
       system "echo '#!/bin/bash' > cloudready.sh"
       system "temp=\"echo 'pkill twm && openbox &' >> /usr/local/etc/X11/xinit/xinitrc\" && echo \"if [[ '$(cat /etc/lsb-release | grep CHROMEOS_ARC_ANDROID_SDK_VERSION)' = '' ]]; then crew install xinit && $temp ; fi\" >> cloudready.sh"
-      system "bash cloudready.sh && rm cloudready.sh"
+      system "bash cloudready.sh"
       puts
       puts "For Chrome OS/Fyde OS user:".lightblue
       puts "To use this package, you need to download XServer XSDL from Google Play Store".lightblue

--- a/packages/openbox.rb
+++ b/packages/openbox.rb
@@ -20,7 +20,10 @@ class Openbox < Package
   depends_on 'glib'
   depends_on 'libxfixes'
   depends_on 'libsm'
-  depends_on 'pango' => :build
+  depends_on 'pango'
+  depends_on 'pangomm'
+  depends_on 'graphite'
+  depends_on 'harfbuzz'
   depends_on 'libx11' => :build
   depends_on 'adwaita_icon_theme'
   depends_on 'xdpyinfo'

--- a/packages/opera.rb
+++ b/packages/opera.rb
@@ -12,6 +12,8 @@ class Opera < Package
     depends_on 'alien' => :build
     depends_on 'gtk3'
     depends_on 'gsettings_desktop_schemas'
+    depends_on 'harfbuzz'
+    depends_on 'graphite'
     depends_on 'sommelier'
   end
 

